pkg/plugins/resources/gitlab/branch/main.go

```diff
@@ -37,7 +37,7 @@ type Gitlab struct {
 	versionFilter version.Filter
 }
 
-// New returns a new valid Gitlab object.
+// New returns a new valid GitLab object.
```

Comment:
This is technically a reference to the `Gitlab object`, but for simplicity, I didn't micromanage that correction. I'm happy to drop this change.

Reply:
That's a good point